scout.py

```diff
@@ -102,6 +96,5 @@ def search_results(results_dict, gen):
 if __name__ == '__main__':
     results_dict = {}
     print_scout()
-    for x in censys_search(args.ip):
-        search_results(results_dict, x)
+    map(search_results, results_dict, censys_search(args.ip)
```

Comment:
Fixing  '''skid code'''

> map(search_results, results_dict, censys_search(args.ip))

Disclaimer: missing bracket was not identified by the author